### ISSUE TITLE
NoClassDefFoundError on 1.18.1 dedicated server

### ISSUE THREAD

**Issue by copygirl** [glisco03/victus#6]
I'm aware this could well be because the mod hasn't been updated
to 1.18.1 yet, but I did want to report it in case you were not aware.

### Versions

- Minecraft `1.18.1`
- Victus `0.1.4+1.18`
- More mods.. let me know if they're relevant.

### Log
```
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_437
        at com.glisco.victus.item.VictusItemGroup.setup(VictusItemGroup.java:18) ~[victus-0.1.4+1.18.jar:?]
        at io.wispforest.owo.itemgroup.OwoItemGroup.initialize(OwoItemGroup.java:75) ~[owo-lib-0.3.10+1.18.jar:?]
        at com.glisco.victus.Victus.onInitialize(Victus.java:48) ~[victus-0.1.4+1.18.jar:?]
        at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.12.12.jar:?]
        ... 16 more
```

**Comment by gliscowo** [comment]
I'm about to fucking cry (not actually). I pushed an update for owo literally right before Christmas which is the first time it ever contained a critical bug. I'll get on fixing it as soon as I can but it has already been reported on three different projects🥲
Thanks for reporting in any case, but this bug is already known. You can fix it by going back one owo version

**Comment by copygirl** [comment]
I know myself about those kind of issues. No worries. Enjoy your Holidays! :green_heart:
I'll just be waiting for Victus and oωo updates to re-add the mod.

Thanks for your work.

**Comment by gliscowo** [comment]
Fixed since 0.3.11, but 0.3.12 should be used as 0.3.11 contains a different bug
